src/containers/Masthead/MastheadContainer.tsx

```diff
@@ -69,23 +75,38 @@ const DrawerWrapper = styled("div", {
 
 const mastheadQuery = gql`
   query mastHead($subjectId: String!) {
-    subject(id: $subjectId) {
-      ...MastheadDrawer_Subject
+    root: node(id: $subjectId) {
+      ...MastheadDrawer_Root
     }
   }
-  ${MastheadDrawer.fragments.subject}
+  ${MastheadDrawer.fragments.root}
 `;
 
 const MastheadContainer = () => {
   const { t, i18n } = useTranslation();
-  const { subjectId } = useUrnIds();
+  const { contextId, subjectId: subId, topicList } = useUrnIds();
   const { user } = useContext(AuthContext);
   const { openAlerts, closeAlert } = useAlerts();
+  const { data: rootData, loading: rootLoading } = useGraphQuery<GQLContextQuery, GQLContextQueryVariables>(
```

Comment:
Hvorfor trenger vi to separate queries her?

Reply:
Spørring en bruker contextId og henter ut parameteret som trengs til spørring to. Når vi flytter gamle urler til en server-redirect kan vi forenkle.

Reply:
Kan vi ikke conditionally kalle på querien med forskjellige variabler basert på om pene URL'er er skrudd på?

Reply:
Sånn vi gjør de andre steder, altså

Reply:
Vi henter litt andre data her fordi vi trenger subjectId for subjectmeny. Så det vil uansett måtte bli to spørringer.